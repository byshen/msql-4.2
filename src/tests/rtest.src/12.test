# Bulk insert testing

create table bulk_test ( v1 int, v2 char(5), v3 date) \p\g

insert into bulk_test values (1, 'A', '1-Jan-2001'), 
	(2, 'B', '2-Jan-2001'), (3, 'C', '3-Jan-2001'), 
	(4, 'D', '4-Jan-2001'), (5, 'E', '5-Jan-2001'), 
	(6, 'F', '6-Jan-2001'), (7, 'G', '7-Jan-2001'), 
	(8, 'H', '8-Jan-2001'), (9, 'I', '9-Jan-2001')
\p\g

select * from bulk_test \p\g
