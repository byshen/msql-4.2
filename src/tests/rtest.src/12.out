
Welcome to the miniSQL monitor.  Type \h for help.


mSQL >     -> 
Query buffer
------------

create table bulk_test ( v1 int, v2 char(5), v3 date) 
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL >     ->     ->     ->     ->     ->     -> 
Query buffer
------------
insert into bulk_test values (1, 'A', '1-Jan-2001'), 
	(2, 'B', '2-Jan-2001'), (3, 'C', '3-Jan-2001'), 
	(4, 'D', '4-Jan-2001'), (5, 'E', '5-Jan-2001'), 
	(6, 'F', '6-Jan-2001'), (7, 'G', '7-Jan-2001'), 
	(8, 'H', '8-Jan-2001'), (9, 'I', '9-Jan-2001')

[continue]
    -> 
Query OK.  9 row(s) modified or retrieved.




mSQL >     -> 
Query buffer
------------
select * from bulk_test 
[continue]
    -> 
Query OK.  9 row(s) modified or retrieved.

 +----------+-------+-------------+
 | v1       | v2    | v3          |
 +----------+-------+-------------+
 | 1        | A     | 01-Jan-2001 |
 | 2        | B     | 02-Jan-2001 |
 | 3        | C     | 03-Jan-2001 |
 | 4        | D     | 04-Jan-2001 |
 | 5        | E     | 05-Jan-2001 |
 | 6        | F     | 06-Jan-2001 |
 | 7        | G     | 07-Jan-2001 |
 | 8        | H     | 08-Jan-2001 |
 | 9        | I     | 09-Jan-2001 |
 +----------+-------+-------------+



mSQL > 
Bye!

